### PR TITLE
fix: headlines should be the same on all native platforms

### PR DESCRIPTION
**Pull request by jmars**
For some reason IOS had different style sheets for headlines than android, editorial have asked that it looks the same on both native platforms so lets use just one native stylesheet.
image

### CODE REVIEW

File: packages/article-magazine-comment/src/styles/index.js [hunk 1]
```diff
@@ -2,19 +2,24 @@ import { StyleSheet } from "react-native";
 import { spacing } from "@times-components/styleguide";
 import nativeStyles from "./native";
 
-const styles = StyleSheet.create({
+const androidStyles = {
```
Comment (nathggns) [inline packages/article-magazine-comment/src/styles/index.js]: Don't these apply on iOS too?

**Comment by parkjh** [comment]
> For some reason IOS had different style sheets for headlines than android, editorial have asked that it looks the same on both native platforms so lets use just one native stylesheet.

please provide screenshots of all the article types that are affected here

File: packages/article-magazine-comment/src/styles/index.js (hunk 1 above)
Comment (parkjh) [reply packages/article-magazine-comment/src/styles/index.js]: I think this is just poor choice of naming...

Comment (parkjh) [inline packages/article-magazine-comment/src/styles/index.js]:
```suggestion
const appStyles = {
```

File: packages/article-magazine-standard/src/styles/index.js [hunk 2]
```diff
@@ -1,8 +1,21 @@
 import { StyleSheet } from "react-native";
+import { spacing } from "@times-components/styleguide";
 import nativeStyles from "./native";
 
+const androidStyles = {
```
Comment (parkjh) [inline packages/article-magazine-standard/src/styles/index.js]:
```suggestion
const appStyles = {
```

File: packages/article-main-standard/src/styles/article-header/index.js [hunk 3]
```diff
@@ -1,13 +1,16 @@
 import { StyleSheet } from "react-native";
-import { spacing } from "@times-components/styleguide";
+import { fontSizes, spacing } from "@times-components/styleguide";
 import globalStyle from "../shared";
 import sharedStyles from "./shared";
 
-const nativeStyles = {
+const androidStyles = {
```
Comment (parkjh) [inline packages/article-main-standard/src/styles/article-header/index.js]:
```suggestion
const appStyles = {
```

File: packages/article-magazine-comment/src/styles/index.js [hunk 4]
```diff
@@ -2,19 +2,24 @@ import { StyleSheet } from "react-native";
...
-  metaSpacer: {
-    marginTop: spacing(2)
+  standFirst: {
+    ...nativeStyles.standFirst,
+    marginBottom: spacing(2.5)
   }
+};
+
+const styles = StyleSheet.create({
+  ...androidStyles
```
Comment (parkjh) [inline packages/article-magazine-comment/src/styles/index.js]:
```suggestion
  ...appStyles
```

File: packages/article-magazine-standard/src/styles/index.js [hunk 5]
```diff
@@ -1,8 +1,21 @@
...
+  },
+  metaTablet: {
+    ...nativeStyles.metaTablet,
+    paddingTop: "auto"
+  }
+};
+
 const styles = StyleSheet.create({
-  ...nativeStyles
+  ...androidStyles
```
Comment (parkjh) [inline packages/article-magazine-standard/src/styles/index.js]:
```suggestion
  ...appStyles
```

File: packages/article-main-standard/src/styles/article-header/index.js [hunk 6]
```diff
@@ -29,7 +34,7 @@ const nativeStyles = {
 
 const styles = StyleSheet.create({
   ...globalStyle,
-  ...nativeStyles
+  ...androidStyles
```
Comment (parkjh) [inline packages/article-main-standard/src/styles/article-header/index.js]:
```suggestion
  ...appStyles
```

**Pull request merged**